### DISCUSSION TITLE
how can add one branch to ignore

### DISCUSSION THREAD

**Discussion by HardCoreQual** [GitLiveApp/GitLive#128] [Q&A]
In my project exist branch `dev` that contains all features that now is in develop

it mean that each developer that want to test how it work on server, will rebase dev branch on feature branch and redeploy it on server

dev branch will never merge in master, and conflicts with dev is not important because if exist serious conficts, can just remove dev branch and recreate it again

I want just add `dev` branch to list of ignored branches for stop show conficts with this branch, but not find how to do it for gitlive in Intellij IDEA

**Comment by Ygnys** [comment]
Hi @HardCoreQual unfortunately atm we don't support this feature but we will add it to our roadmap. Thanks for the suggestion, I will update you once we release it
